packages/core/test/integration/data-types/data-types.test.ts

```diff
@@ -745,7 +768,11 @@ describe('DataTypes', () => {
       });
 
       it('is deserialized as a string when DataType is not specified', async () => {
-        await testSimpleInOutRaw(vars.User, 'bigintAttr', 123n, '123');
+        if (dialect.name !== 'oracle') {
```

Comment:
I think it would be good that this has the same result for Oracle as well

Reply:
Oracle inferences the values from metadata and returns data in respective format. In this case, NUMBER.